tests/unit/package_managers/test_pip.py

```diff
@@ -42,11 +42,6 @@ def setup_module() -> None:
     pip.log.setLevel(logging.DEBUG)
 
 
-@pytest.fixture
```

Comment:
If this were changes to the pip module I'd be fine with it, but this commit feels misplaced in the overall changes and theme set by this PR, so please propose it as a standalone PR. Also, for future reference, when posting a PR, please include fixes/refactors like this one at the very beginning, not in the middle of the patch series, the sequence of patches should follow some logical order, it helps the reviewer not to lose the train of thoughts, just a recommendation ;).

Reply:
I know. As I already responded to Alexey, I rebased on the high level code branch. Otherwise both commits would be at the beginning.